README.md

```diff
@@ -126,6 +126,10 @@ beyond the LTS release.
 </tr>
 </table>
 
+## A note about v4.2 LTS
+
+While v4.2 was released in October, *this is a one-off event!* There was no LTS release plan in place in April 2015; v4.2 LTS was released in August in order to get v4.x updates into an LTS release as quickly as possible. In the future **LTS releases will be cut in April**.
```

Comment:
This isn't actually correct.

The release will be the version cut in April but not designated as LTS or in LTS mode until October.